README.md

```diff
@@ -1,3 +1,10 @@
+Upgrade Notice
+==============
+
+Miners and users of namecoind should switch to [Namecoin Core](github.com/namecoin/namecore), a reimplementation of the Namecoin protocol that maintains parity with Bitcoin.
```

Comment:
Mention that this only applies to users who can build from source.  Also mention that this only applies to users who do not use Namecoin-Qt (i.e. it doesn't apply to users who use both namecoind and Namecoin-Qt).